### ISSUE TITLE
flipping the watchdog does not seem to work anymore

### ISSUE THREAD

**Issue by romor** [sfera-labs/strato-pi-kernel-module#2]
Up to now we toggled the watchdog by writing `F` to `/sys/class/stratopi/watchdog/heartbeat`. This does not seem to work anymore and the watchdog gets triggered.

We changed it and write `0` and `1` now to the heartbeat as a work-around. This works as expected and the watchdog does not trigger during normal operation.

**Comment by giampiero7** [comment]
Hi @romor ,
What do you see if you run the following?

```
cat /sys/class/stratopi/watchdog/heartbeat
echo F > /sys/class/stratopi/watchdog/heartbeat
cat /sys/class/stratopi/watchdog/heartbeat
```

**Comment by romor** [comment]
```
$ cat /sys/class/stratopi/watchdog/heartbeat
0
$ echo F > /sys/class/stratopi/watchdog/heartbeat
$ cat /sys/class/stratopi/watchdog/heartbeat
0
$ echo 1 > /sys/class/stratopi/watchdog/heartbeat
$ cat /sys/class/stratopi/watchdog/heartbeat
1
$ echo 0 > /sys/class/stratopi/watchdog/heartbeat
$ cat /sys/class/stratopi/watchdog/heartbeat
0
$ echo G > /sys/class/stratopi/watchdog/heartbeat
-bash: echo: write error: Invalid argument
$ echo F > /sys/class/stratopi/watchdog/heartbeat
$ cat /sys/class/stratopi/watchdog/heartbeat
0
$ uname -a
Linux embyt-strato 6.1.19-v7+ #1637 SMP Tue Mar 14 11:04:52 GMT 2023 armv7l GNU/Linux
```

**Comment by giampiero7** [comment]
Thanks for the report.
The kernel function `kstrtobool()` has been modified in kernel version 6.1 changing behaviour... don't know how this is acceptable.
Anyway, fixed in the last update (v1.14).
